README.md

```diff
@@ -1 +1,42 @@
-# fliplet-widget-interactive-floorplan
\ No newline at end of file
+# Fliplet Interactive Floorplan
```

Comment:
As discussed in https://github.com/Fliplet/fliplet-studio/issues/3736 we would call this Interactive Map. Although the name and package name is changed, there's still A LOT of references to floor plan. Something that's only going to cause confusion and further inconsistencies if we start naming parts of new feature using naming convention related to `interactive map`.

I'd swap them all out before the feature goes on production.

Reply:
Case in point

![image](https://user-images.githubusercontent.com/290733/52480731-73445600-2ba4-11e9-8847-cb33b542f164.png)